### ISSUE TITLE
Giving weights to shifts

### ISSUE THREAD

**Issue by gelazari** [Tester2012/or-tools-scripts#1]
Hi Anatolii, this is @azal from StackOverflow. Thanks again for the solution. I have the following question: 

Let's assume that these drivers are part of a delivery company where each slot has a specific demand of parcels to be delivered. I was wondering if it is possible to somehow incorporate the demand at each slot and thus, optimize the schedule and the breaks given the weights of the slots. This is something that I believe would make more sense to a real life schedule. 

For instance, if we have 3 drivers then there are many possible solutions for a schedule with breaks during the range [07:00, 24:00]. But if we incorporate the demand per slot, then we force the solver somehow to create schedule and breaks in order to cover the demand as good as possible. What do you think?

**Comment by Tester2012** [comment]
@gelazari are you saying that working shifts may require different number of driving drivers, right? For example, you may need only `1` driver from 7:00 to 11:00 and at least `2` from 14:00 to 16:00.

**Comment by gelazari** [comment]
Yes. But the breaks should also not overlap given that. In my kind I was thinking something like assigning weights to each shift and trying to assign breaks at the shifts with the lower weights and if breaks overlap then change the overlapping breaks as well
